src/pages/workspaces/workspace/Data.js

```diff
@@ -969,6 +1025,13 @@ const WorkspaceData = _.flow(
               await loadMetadata()
               setSelectedData({ type: workspaceDataTypes.entities, entityType: tableName })
             })
+          })],
+          [workspaceDataTypes.wds, () => h(WDSContent, {
+            key: refreshKey,
+            workspaceUUID: workspaceId,
+            workspace,
+            recordType: selectedData.entityType,
+            wdsSchema
           })]
```

Comment:
consideration for future:
this is a pretty big file.  I would recommend starting to split this off into sub-components.... and aligning the data-call need with useInterestingData hook pattern that I paired with Nick on and will be promoting for broad usage.  That will make unit testing this code way easier (which we are starting to push on harder and harder for PRs)